clang/test/dpct/nccl.cu

```diff
@@ -6,6 +6,10 @@
 
 int check(ncclResult_t);
 
+// CHECK: #if DPCT_CCL_VERSION >= 0
+#if NCCL_VERSION_CODE >= NCCL_VERSION(2,13,0)
```

Comment:
current NCCL_VERSION(X,Y,Z) it migrate to zero,  maybe not make sense in some case,  how about migrate to "DPCT_CCL_VERSION" an non zero value.